File: 8kyu/calculate-bmi/calculate-bmi.py
# def bmi(weight, height):
#   bmi = round(weight / height ** 2, 1)
#   if bmi <= 18.5:
#     return "Underweight"
#   if bmi <= 25.0:
#     return "Normal"
#   if bmi <= 30.0:
#     return "Overweight"
#   if bmi > 30.0:
#     return "Obese"

def bmi(weight, height):
  bmi = round(weight / height ** 2, 1)
  result = ["Underweight", "Normal", "Overweight", "Obese"]
  return result[(bmi > 30) + (bmi > 25) + (bmi > 18.5)]


print(bmi(50, 1.80))
print(bmi(80, 1.80))
print(bmi(90, 1.80))

File: 8kyu/calculate-bmi/calculate-bmi.js
/* 
PREP:
- calc BMI based on weight / height ** 2
- use if statement to return the result
*/

function bmi(weight, height) {
	const calcBMI = weight / height ** 2;
	if (calcBMI <= 18.6) {
		return "Underweight";
	}
	if (calcBMI <= 25) {
		return "Normal";
	}
	if (calcBMI <= 30) {
		return "Overweight";
	}
	if (calcBMI > 30) {
		return "Obese";
	}
}
